### PR TITLE
Remove binding redirect of System.Collections.Immutable

### PR DESCRIPTION
**Pull request by troydai**
To prevent assembly load loop in DNX.

@bricelam @davidfowl

### CODE REVIEW

**Comment by lajones** [comment]
You can probably just remove the files altogether if we no longer need the binding redirect.

File: test/EntityFramework.Commands.FunctionalTests/App.config [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a"/>
-        <bindingRedirect oldVersion="1.1.36.0" newVersion="1.1.37.0"/>
```
Comment (divega) [inline test/EntityFramework.Commands.FunctionalTests/App.config]: Does the assemblyBinding element have any purpose after removing the bindingRedirect?

**Comment by troydai** [comment]
I'll remove the entire file.

**Comment by davidfowl** [comment]
Those files never worked until @anurse 's checkin yesterday which is causing problems.

**Comment by troydai** [comment]
:watch:

**Comment by troydai** [comment]
I'll merge soon to unblock the CI.

**Comment by divega** [comment]
:shipit: to unblock the CI. I assume these were added to workaround some issue. If the issue comes back we can revisit.

**Comment by natemcmaster** [comment]
We needed those to run tests in Visual Studio. Most of the team uses TestDriven.NET.

**Pull request merged**